### PR TITLE
Answered 'How to extend Carefree products which expired beyond the grace period'

### PR DESCRIPTION
**Pull request by fiskaltrust-MBI**

### CODE REVIEW

File: qna/carefree-extension-after-expired-grace-period.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Question
+
+How to extend Carefree products which expired beyond the grace period?
+
+## Metadata tags
+
+lang-en, market-at, bundle-carefree, portal, cashbox management, PosOperator, PosDealer, Consultant, support-required
```
Comment (christian-rogobete) [inline qna/carefree-extension-after-expired-grace-period.md:7]: Tag name "support-required" is not part of the tag list (see landing page of the repo). Can you please remove it

Comment (AxelKutschera) [reply qna/carefree-extension-after-expired-grace-period.md:7]: The file-name should be how described in the landing page with starting eg "AT-":
"AT-carefree-extension-after-expired-grace-period.md"

Comment (fiskaltrust-MBI) [reply qna/carefree-extension-after-expired-grace-period.md]: > The file-name should be how described in the landing page with starting eg "AT-":
> "AT-carefree-extension-after-expired-grace-period.md"

The file has been renamed. Thanks for the suggestion!

Comment (fiskaltrust-MBI) [reply qna/carefree-extension-after-expired-grace-period.md]: > Tag name "support-required" is not part of the tag list (see landing page of the repo). Can you please remove it

The tag has been removed. Thank you for spotting it!

**Pull request merged**